pkg/events/logger_handler/logger_handler.go

```diff
@@ -0,0 +1,101 @@
+package logger_handler
+
+import (
+	k8sV1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+
+	"github.com/submariner-io/admiral/pkg/log"
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/events"
+)
+
+type LoggerHandler struct {
+	events.EventHandlerBase
+}
+
+// GetName returns the name of the event handler
+func (l *LoggerHandler) GetName() string {
+	return "logger"
+}
+
+// GetNetworkPlugin ... logging driver says: "all"
+func (l *LoggerHandler) GetNetworkPlugin() string {
+	return events.AnyNetworkPlugin
+}
+
+// TransitionToNonGateway is called when there is a transition from GatewayNode to NonGateway on the
+// This will be called only when the transition happens and not periodically.
+func (l *LoggerHandler) TransitionToNonGateway() error {
+	klog.V(log.DEBUG).Info("The current node stopped being Gateway")
+	return nil
+}
+
+// TransitionToGateway is called when there is a transition from NonGatewayNode to GatewayNode.
+// This will be called only when the transition happens and not periodically.
+func (l *LoggerHandler) TransitionToGateway() error {
+	klog.V(log.DEBUG).Info("The current node became an active Gateway")
+	return nil
+}
+
+// LocalEndpointCreated is called when the endpoint for the localCluster is created.
+func (l *LoggerHandler) LocalEndpointCreated(endpoint submV1.Endpoint) error {
+	klog.V(log.DEBUG).Infof("A new local Endpoint has been discovered for active Gateway on node %q with IP %q",
+		endpoint.Spec.Hostname, endpoint.Spec.PrivateIP)
```

Comment:
```suggestion
	klog.V(log.DEBUG).Infof("A new Endpoint for the local cluster has been created: %#v", endpoint)
```

Reply:
We didn't want to make it overly verbose, but .. honestly it's not an event that happens a lot